geopandas/io/file.py

```diff
@@ -116,8 +116,17 @@ def to_file(df, filename, driver="ESRI Shapefile", schema=None, **kwargs):
     if schema is None:
         schema = infer_schema(df)
     with fiona_env():
+        crs_wkt = None
+        try:
+            gdal_version = fiona.env.get_gdal_release_name()
+        except AttributeError:
+            gdal_version = "2.0.0"  # just assume it is not the latest
+        if LooseVersion(gdal_version) >= LooseVersion("3.0.0") and df.crs:
+            crs_wkt = df.crs.to_wkt()
+        elif df.crs:
+            crs_wkt = df.crs.to_wkt("WKT1_GDAL")
```

Comment:
Should we prefer "EPSG:..." over a WKT *if* available with 100% confidence?

Reply:
I don't think that there is any added value added in doing so.